### ISSUE TITLE
Update redigo pacakage version

### ISSUE THREAD

**Issue by vibyird** [RichardKnop/machinery#704]
Redigo package with v2.0.0 version does not use go mod, but its version is higher than version which use go mod, such as  package with v1.8.5 version. When one project use redigo and use machinery at the same time, go will choose package with v2.0.0 version, not package with v1.8.5 version. because its version is highest. But is is  actually the oldest package.

**Comment by wantdrink** [comment]
yeah I saw the warning when execute go list as below. I'm using v1.10.6.

`go list -m: github.com/gomodule/redigo@v2.0.0+incompatible: retracted by module author: Old development version not maintained or published.`
